R/convertDiscrete.R

```diff
@@ -65,6 +65,6 @@ discreteValueToName = function(par, x) {
   if (par$type == "discrete") {
     ns[getIndex(par$values, x)]
   } else if (par$type == "discretevector")  {
-    sapply(x, function(x) ns[getIndex(x, values = par$values)])
+    vcapply(x, function(app  x) ns[getIndex(x, values = par$values)])
```

Comment:
The nice thing about `sapply` is that it gives `list()` in case of `length(x) == 0`, which would match quite nicely with the *actual default* being `list()`. When [the related BBmisc issue](https://github.com/berndbischl/BBmisc/issues/74) gets fixed, this will result in the most natural printout. `vcapply` will give `character(0)` when `x` is `list()`, which is a deviation from the behaviour of the other "vector" types (which always show `logical(0)`, `numeric(0)` etc. depending on their type).

Not going to say much about the syntax error here.